cypress/integration/project_source_browsing.spec.js

```diff
@@ -6,7 +6,7 @@ before(() => {
 
 beforeEach(() => {
   cy.visit("./public/index.html#/profile/projects");
-  cy.contains("Monadic").click();
+  cy.contains("platinum").click();
```

Comment:
Did we change the fixtures?

Reply:
I did. We created a project called "Monadic" before, which would make more sense for an org, so I changed it to something that resembles a project.

Reply:
Thanks for the clarification.